crates/mipsy/Cargo.toml

```diff
@@ -11,15 +12,10 @@ mipsy_lib = { version = "0.1.0", path = "../mipsy_lib" }
 mipsy_parser = { version = "0.1.0", path = "../mipsy_parser" }
 mipsy_interactive = { version = "0.1.0", path = "../mipsy_interactive" }
 mipsy_utils = { version = "0.1.0", path = "../mipsy_utils" }
-mipsy_instructions = { path = "../mipsy_instructions", features = ["rt_yaml"] }
-clap = { version = "3.1.6", features = ["derive"] } # cli arg parsing
-rustyline = "9.1.1"         # readline support for interactive mipsy
```

Comment:
where did these dependancies go? :thinking: 

Or are they supposed to be in mipsy_interactive instead, and hence these are useless?

Reply:
These are mipsy_interactive dependencies. They aren't used by this crate.